components/raftstore/src/store/fsm/peer.rs

```diff
@@ -546,6 +546,7 @@ where
                     }
                 }
                 PeerMsg::RaftCommand(cmd) => {
+                    self.ctx.disk_full_opt = cmd.disk_full_opt;
```

Comment:
`disk_full_opt` works on every commnad instead of a batch of commands. So it shouldn't be put into `PollContext`.

Reply:
Done!